src/core/tokenHandler.js

```diff
@@ -106,64 +107,58 @@ export default function tokenHandlerFactory(options) {
          * @param {String} newToken
          * @returns {Promise<Boolean>} - resolves true if successful
          */
-        setToken: function setToken(newToken) {
+        setToken(newToken) {
             return tokenStore.enqueue(newToken);
         },
 
         /**
          * Extracts tokens from the Client Config which should be received on every page load
          * @returns {Promise<Boolean>} - resolves true when completed
          */
-        getClientConfigTokens: function getClientConfigTokens() {
-            var self = this;
-            var clientTokens = _.map(module.config().tokens, function(serverToken) {
-                return {
-                    value: serverToken,
-                    receivedAt: Date.now()
-                };
-            });
+        getClientConfigTokens() {
+            const clientTokens = _.map(module.config().tokens, serverToken => ({
+                value: serverToken,
+                receivedAt: Date.now()
+            }));
 
             // Record that this function ran:
             clientConfigFetched = true;
 
-            return Promise.resolve(clientTokens).then(function(newTokens) {
-                // Add the fetched tokens to the store
-                // Uses a promiseQueue to ensure synchronous adding
-                var setTokenQueue = promiseQueue();
+            return Promise.resolve(clientTokens)
+                .then(newTokens => {
+                    // Add the fetched tokens to the store
+                    // Uses a promiseQueue to ensure synchronous adding
+                    const setTokenQueue = promiseQueue();
 
-                _.forEach(newTokens, function(token) {
-                    setTokenQueue.serie(function() {
-                        return self.setToken(token);
+                    _.forEach(newTokens, token => {
```

Comment:
Same as above with Array.forEach()

Reply:
Indeed, as we control the data type, this is actually an array